### ISSUE TITLE
Document input format for all KPLs + Models

### ISSUE THREAD

**Issue by ianstenbit** [keras-team/keras-cv#1343]
Every model + KPL needs to include in its docstring information about the expected dictionary input format.

**Comment by soma2000-lang** [comment]
@ianstenbit Is this issue available for community contributions?If yes then I would love to contribute

**Comment by ianstenbit** [comment]
@soma2000-lang yes this is.

I will write up a docstring to describe our input format which can be included in Layers and models. I'll post it here this afternoon and then add the contribution-welcome label.

**Comment by ianstenbit** [comment]
For community members who'd like to update docstrings for this issue, please use (or adapt) the language in the docstring update in #1363

**Comment by soma2000-lang** [comment]
@ianstenbit Probably I can work on this one.

**Comment by atharvapurdue** [comment]
Hi @soma2000-lang  @ianstenbit , is this issue open? Do you mind if I contribute to it as well? I am new to this repo and this has the 'good first issue' tag.  Thanks!

**Comment by soma2000-lang** [comment]
@atharvapurdue I am working on adding doctrines on few of the KPLs ,but there are many where doctrings needs to

**Comment by YimjuneLiu** [comment]
Hi @ianstenbit @soma2000-lang , Is this issue open? I am still a beginner, do you mind if I participate? Thanks!

**Comment by ianstenbit** [comment]
Yes this issue is open for contributions -- it's a good issue for a beginner 👍 

Feel free to open a PR and tag this issue.

**Comment by YimjuneLiu** [comment]
> Yes this issue is open for contributions -- it's a good issue for a beginner 👍 
> 
> Feel free to open a PR and tag this issue.

Thanks for your reply.

I see that some contributors have already done some work. Do I still need to do all the work? Or do part of it to prevent conflicts? If only part of it is done, which part specifically?

**Comment by soma2000-lang** [comment]
@YimjuneLiu  Yes ,I am working on some of  the files .It would be great if you work on the other remaining KPL

**Comment by YimjuneLiu** [comment]
> @YimjuneLiu  Yes ,I am working on some of  the files .It would be great if you work on the other remaining KPL

Ok, Can you tell me specifically which KPLs are included?

**Comment by YimjuneLiu** [comment]
#1467

**Comment by Anirban0011** [comment]
@ianstenbit Hi, is this issue still open to contributions? I would like to participate if it is the case. Thanks

**Comment by YimjuneLiu** [comment]
>  @ianstenbit Hi, is this issue still open to contributions? I would like to participate if it is the case. Thanks

Go ahead, I have done some docstrings of KPLs,  you can view the modified files through the pull request above, and then you can modify the remaining files.

**Comment by Anirban0011** [comment]
I have added a PR for docstring on a KPL file, not sure if the format fits, if it is fine, i will create PRs on the remaining files.
#1521

**Comment by Anirban0011** [comment]
#1521

**Comment by Anirban0011** [comment]
#1521 Update some docstrings for few KPLs

**Comment by sqali** [comment]
Hi @ianstenbit ,

Hope you are doing well. Kindly confirm if any of the classes in the below file (keras-cv/benchmarks/vectorization_strategy_benchmark.py) require docstring functions such as:

https://github.com/keras-team/keras-cv/blob/0cff8e2356fb68888b035a7232e904097d3fad46/benchmarks/vectorization_strategy_benchmark.py#L113-L117

https://github.com/keras-team/keras-cv/blob/0cff8e2356fb68888b035a7232e904097d3fad46/benchmarks/vectorization_strategy_benchmark.py#L293-L300

**Comment by man0045** [comment]
is this issue still open to contribute i love to contribute in it if possible?

**Comment by y-vectorfield** [comment]
@ianstenbit, is this issue still active?
I would like to contribte this.

**Comment by frostbyte012** [comment]
@ianstenbit  @LukeWood I also want to contribute to this issue. Is this still open?

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new **KerasHub** package, which supports image, text, and multi-modal models. Please read [the announcement](https://github.com/keras-team/keras-hub/issues/1831). `KerasHub` will support all the core functionality of `KerasCV`.

`KerasHub` can be installed with `!pip install -U keras-hub`. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).
